src/compiler/generator.js

```diff
@@ -77,6 +77,22 @@ const generator = (node) => {
         value: `${stylePrefix}${node.props.bullet} ${renderedChildren.value}\n`,
         styles: renderedChildren.styles
       };
+    case 'img':
+      const { src, width, height, style } = node.props
+      const w = style && style.width ? parseInt(style.width, 10) : width
+      const h = style && style.height ? parseInt(style.height, 10) : height
+      return {
+        value: `\n${stylePrefix}+\n`,
+        styles: {
+          background: `url(${src})`,
+          backgroundSize: `${w} ${h}`,
+          color: 'transparent',
```

Comment:
What about adding `color` and `fontSize` (which are static) in the `transformer.js` file as `defaultProps` ?

Reply:
I started to making this way but I found myself having to tweak in lots of places to handle the `node` without `children`. 
Let me know what you think could be a better way. As I understand, the `transformer`/`traverser` functions assumes that each `node` should have a `children` or an `Array` of `children`.
Happy to update the code in the best way for the _library_ logic.

Reply:
It's not a big deal I think. We are still going to have styles in here, so I'm ok on leaving them all here